users/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views           import View
+from django.http            import JsonResponse
+
+from users.models           import User
+from suulgo.settings        import SECRET_KEY, ALGORITHM
+
+class LoginInfoView(View):
```

Comment:
유저의 프로필 정보를 보는 VIew이기 때문에 간단하게 ProfileView로 작성할 수 있을 것 같아요~
```suggestion
class ProfileView(View):
```

Reply:
넵 수정하겠습니다!